LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License
```

Comment:
2 different licences in here https://github.com/plan3/hapi-common-auth/blob/master/package.json#L21